### ISSUE TITLE
Allow time to see the image you have to count BEFORE hack starts fully or give 2-3 number leeway. 

### ISSUE THREAD

**Issue by TeammateBlamer** [echoroleplay/feedback#241]
**Suggestion Description**
When doing the hack for the back gate inside fleecas, you are given a shape and must count the shape for 90 seconds as up to 5-7 shapes all appear. This hack is very very difficult because before you even see what image youre looking for 2-3 of them can go past.
You also need to get the exact number. If the answer is 88 for example, if you were to guess 87 the hack would fail. If you were given 5 seconds to see what shape you had to count before the images started to appear OR if you could complete the hack by getting 2-3 in range of the number it would be a lot better (if the answer is 88 and you guess 86 you would still be successful)

**Reason**
Crime is in a pretty bad place right now, not many people have access to boosting, Very little thermite being given out for vangies, and people arent doing banks because of the difficulty of the hacks. This is making a lot of the gangs bored as well as a lot of the cops. This would be a small step to improving the health of the criminal world right now.

**Additional context**
NA

**Comment by Creeamy** [comment]
I agree with letting people see the shape for a few seconds before it starts but I think it should stay at getting the exact number correct. People are not doing fleecas because of access to the tools to hack it not because of the hacks themselves.

**Comment by Lane2G** [comment]
The main issue I have with it is the 90 seconds, But +1 to seeing the shape and i think it should stay exact but maybe 50 seconds.
